homeassistant/components/dunehd/media_player.py

```diff
@@ -106,11 +132,6 @@ def is_volume_muted(self):
         """Return a boolean if volume is currently muted."""
         return int(self._state.get("playback_mute", 0)) == 1
 
-    @property
-    def source_list(self):
```

Comment:
It seems like there is an api to discover these?

https://dune-hd.com/support/ip_control/dune_ip_control_overview.txt

Reply:
@bdraco What exactly do you mean?
This? `http://10.0.0.1/cgi-bin/do?cmd=open_path&url=root://sources`. This should open the `root://sources` folder on the TV screen instead of returning the sources list.
The component used such `sources`:
```yaml
sources:
    Movie1: "smb://10.0.0.1/media/movies/movie1.mkv"
    Movie2: "smb://10.0.0.1/media/movies/movies2.mkv"
```
When the user selects the "Movie1" source, the player reports that the "/tmp/mnt/smb/0/media/movies/movies1.mkv" file is being played. It's a mess and we don't have a way to validate these sources.

Reply:
Thanks for explaining.  I agree, it makes sense to remove that.  It should really be handled by a service.